packages/utilities/psammead-assets/src/__snapshots__/svgs.test.jsx.snap

```diff
@@ -117,6 +117,33 @@ exports[`arabic SVG should render correctly 1`] = `
 </svg>
 `;
 
+exports[`archive SVG should render correctly 1`] = `
+<svg
+  viewBox="207 24"
+>
+  <g
+    fill-rule="evenodd"
+    stroke="#fff"
+    stroke-width=".335"
+    style="stroke: #fff;"
```

Comment:
Looking at the accessibility of the SVG - when viewing with Firefox's colours being changed, the styling looks good - the outline of the SVG is correctly visible and fill/background colours also good too.
<img width="390" alt="Screenshot of SVG in Firefox" src="https://user-images.githubusercontent.com/3028997/69621358-dc636000-1036-11ea-861a-a02e6843e368.png">